src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/binary_convolution/binary_convolution_kernel_1x1.cpp

```diff
@@ -93,10 +93,10 @@ JitConstants BinaryConvolutionKernel1x1::GetJitConstants(const binary_convolutio
         jit.AddConstant(MakeJitConstant("FILTER_MASK",
                                         (0xFFFFFFFF >> (ic_pack_size - params.inputs[0].Feature().v % ic_pack_size))));
     }
-
     if (params.outputs[0].GetDType() == Datatype::BINARY) {
         jit.AddConstant(MakeJitConstant("BINARY_PACKED_OUTPUT", 1));
     }
+    jit.AddConstant(MakeJitConstant("DG2", params.engineInfo.deviceType == dev_type::discrete_gpu));
```

Comment:
1. "DG2" is not suitable name for the macro.
2. You enable WA for ALL discrete GPUs which may be unexpected in some cases (DG1). So consider changing this check to `engineInfo.bIMMADSupport == true` or something like that

Reply:
1. Removed "DG2" define during code cleaning up.
2. Fixed.